staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery.go

```diff
@@ -477,16 +466,11 @@ func (dm *discoveryManager) RemoveAPIService(apiServiceName string) {
 	}
 }
 
-func (dm *discoveryManager) ExternalServicesSynced() bool {
-	dm.servicesLock.RLock()
-	defer dm.servicesLock.RUnlock()
-	for _, info := range dm.apiServices {
-		if info.lastReconciled.IsZero() {
-			return false
-		}
-	}
-
-	return true
+// Intended Used for testing. Returns true if the queue of services to sync is
```

Comment:
if it is for testing can you make it not exported and/or move it to live in the _test.go file?

Reply:
That would require making the tests whitebox. Wasnt sure of the correct side of the tradeoff to take. Ill change it over then

Reply:
that's unfortunate but I don't think you want external callers of this as it is.

Reply:
I'd rather have the test use an internal method than a function exported for the sake of an external test?

Reply:
correct. im moving it over

Reply:
I've removed `AllServicesSynced` and replaced with internal test function